frontend/src/pages/Movie.tsx

```diff
@@ -25,6 +26,8 @@ import { NewReviewModal } from "../modals/NewReviewModal";
 export const MoviePage = () => {
   const { id } = useParams();
 
+  const [showReviews, setShowReviews] = useState(false);
```

Comment:
Not really understanding why we want to hide the reviews. Might be a better idea to limit them and then add a "Show more" button that takes you to the reviews page